### PR TITLE
[TESTS] No need for < 6 check, as we don't test < L6 anymore

### PR DESCRIPTION
**Pull request by mfn**

### CODE REVIEW

File: tests/Console/ModelsCommand/AbstractModelsCommand.php [hunk 1]
```diff
@@ -6,10 +6,8 @@
 
 use Barryvdh\LaravelIdeHelper\IdeHelperServiceProvider;
 use Barryvdh\LaravelIdeHelper\Tests\SnapshotPhpDriver;
-use Barryvdh\LaravelIdeHelper\Tests\SnapshotTxtDriver;
```
Comment (mfn) [inline tests/Console/ModelsCommand/AbstractModelsCommand.php:9]: Drive-by clean-up, as it's not used (anymore 🤷‍♀️)

**Review by barryvdh — APPROVED** [review]
